components/nimbus/ios/Nimbus/FeatureHolder.swift

```diff
@@ -121,10 +127,33 @@ public class FeatureHolder<T> {
     /// This changes the mapping between a `Variables` and the feature configuration object.
     ///
     /// This is most likely useful during testing and other generated code.
-    public func with(initializer: @escaping (Variables) -> T) {
+    public func with(initializer: @escaping (Variables, UserDefaults?) -> T) {
         lock.lock()
         defer { self.lock.unlock() }
         cachedValue = nil
         create = initializer
     }
 }
+
+/// A bare-bones interface for the FML generated objects.
+public protocol FMLObjectInterface {}
+
+/// A bare-bones interface for the FML generated features.
+///
+/// App developers should use the generated concrete classes, which
+/// implement this interface.
+///
+public protocol FMLFeatureInterface: FMLObjectInterface {
+    /// A test if the feature configuration has been modified somehow, invalidating any experiment
+    /// that uses it.
+    ///
+    /// This may be `true` if a `pref-key` has been set in the feature manifest and the user has
+    /// set that preference.
+    func isModified() -> Bool
+}
+
+public extension FMLFeatureInterface {
+    func isModified() -> Bool {
```

Comment:
Adding `public` gives:

⚠️ `public` modifier is redundant for instance method declared in a public extension.